game/src/environment/LevelMap/Tutorial.tscn

```diff
@@ -1,7 +1,8 @@
-[gd_scene load_steps=5 format=2]
+[gd_scene load_steps=6 format=2]
 
 [ext_resource path="res://src/environment/LevelMap/core/LevelMap.tscn" type="PackedScene" id=1]
 [ext_resource path="res://src/environment/traps/Trap.tscn" type="PackedScene" id=2]
+[ext_resource path="res://src/ui/Menu/Fog.tscn" type="PackedScene" id=3]
```

Comment:
Do u wanna move Fog.tscn somewhere else or duplicate and put a copy in env/map/Fog.tscn? I think duplicating the file in this case q okay, makes sense too in case we have custom options for map.

Reply:
yeah we can make dup it since each fog can be diff color and diff transparency